### PR TITLE
open nav.tum.sexy to show location

### LINKED ISSUE #1437: Use OpenStreetMaps for Roomfinder
I have seen [this app](https://play.google.com/store/apps/details?id=de.unigoettingen.lageplan) and would like to work on this. 
My idea would be to use something like [this](https://github.com/CartoDB/mobile-sdk) to solve the problem.

### PR DESCRIPTION
**Pull request by kmodexc**
## Issue

This fixes the following issue(s):

This only opens nav.tum.sexy and does not have an in-app map. However i think that ok for now and solves the issue for me. 

On step for #1437

### CODE REVIEW

**Comment by joschahenningsen** [comment]
As long as the in app roomfinder does not work this seems to be a reasonable way to go 👌
@CommanderStorm are you fine with this? Will the API URL this way?

**Comment by CommanderStorm** [comment]
Obviously yes.
We should however not keep this as a permanent fix (which should also be obvious)

@kmodexc is working on that too, so I see no problem in merging this

**Comment by kordianbruck** [comment]
I'd probably recommend to instead do this in an in-app webview instead of opening an external browser: https://developer.android.com/guide/webapps/webview

**Comment by kmodexc** [comment]
@kordianbruck Good idea. i will do this

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/NavigatumActivity.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package de.tum.`in`.tumcampusapp.component.tumui.calendar
+
+import android.annotation.SuppressLint
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.webkit.WebView
+
+class NavigatumActivity : AppCompatActivity() {
```
Comment (CommanderStorm) [inline app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/NavigatumActivity.kt:8]:
```suggestion
class NavigaTUMActivity : AppCompatActivity() {
```

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarDetailsFragment.kt [hunk 2]
```diff
@@ -188,10 +187,10 @@ class CalendarDetailsFragment : RoundedBottomSheetDialogFragment() {
     }
 
     private fun onLocationClicked(location: String) {
-        val findStudyRoomIntent = Intent()
-        findStudyRoomIntent.putExtra(SearchManager.QUERY, Utils.extractRoomNumberFromLocation(location))
-        findStudyRoomIntent.setClass(requireContext(), RoomFinderActivity::class.java)
-        startActivity(findStudyRoomIntent)
+        val sendIntent: Intent = Intent(getContext(),NavigatumActivity::class.java).apply {
```
Comment (CommanderStorm) [inline app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarDetailsFragment.kt:190]:
```suggestion
        val sendIntent: Intent = Intent(getContext(),NavigaTUMActivity::class.java).apply {
```

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -371,6 +371,12 @@
                 android:value=".component.ui.overview.MainActivity" />
         </activity>
 
+        <activity
+            android:name=".component.tumui.calendar.NavigatumActivity"
+            android:label="Navigatum"
```
Comment (CommanderStorm) [inline app/src/main/AndroidManifest.xml:376]:
```suggestion
            android:name=".component.tumui.calendar.NavigaTUMActivity"
            android:label="NavigaTUM"
```

**Review by CommanderStorm — COMMENTED** [review]
I dont have a strong oppinion on these suggestions. These are suggestions, not demands.

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/NavigatumActivity.kt (hunk 1 above)
Comment (CommanderStorm) [reply app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/NavigatumActivity.kt:8]: `app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/NavigatumActivity.kt` could be changed also, if you decide that these suggenstions are valid

**Review by joschahenningsen — APPROVED** [review]
Looks good to me. Happy to work on #1451 if no one picked it up already :)

**Pull request merged**